syft/format/internal/spdxutil/helpers/download_location.go

```diff
@@ -1,10 +1,29 @@
 package helpers
 
-import "github.com/anchore/syft/syft/pkg"
+import (
+	"github.com/anchore/syft/syft/pkg"
+	urilib "github.com/spdx/gordf/uri"
+)
 
 const NONE = "NONE"
 const NOASSERTION = "NOASSERTION"
 
+func isUriValid(uri string) bool {
+	_, err := urilib.NewURIRef(uri)
+	return err == nil
+}
+
+func checkUri(uri string) string {
```

Comment:
nit: I think this could be renamed to be slightly more representative of what it's doing, maybe `UriValue`, since it's not "checking" but rather returning the value we should be using.

Reply:
Updated 👍